nixos/tests/mysql-backup.nix

```diff
@@ -23,17 +23,24 @@ import ./make-test.nix ({ pkgs, ... } : {
   testScript =
     '' startAll;
 
+       # delete backup file that may be left over from a previous run
+       $master->execute("rm -f /var/backup/mysql/testdb.gz");
```

Comment:
In case the service run just before the test or when does it happen?

Reply:
This is not necessary on Hydra. It's useful for repeated test runs on a local machine using `nixos-test-driver` because the disk image isn't rebuilt in that case and the file persists.